src/main/resources/messages.properties

```diff
@@ -246,6 +246,7 @@ help-needed.food=Help with food
 help-needed.children=Help with money for children
```

Comment:
It won't let me mark the right spot, so I'll put this here.  If this PR includes the help needed page, then the message for `help-needed.subheader` needs to be updated as it no longer matches the design.